db/primary_migrate/20220406155240_drop_users_sign_in_at_sign_in_ip_sign_in_count.rb

```diff
@@ -0,0 +1,11 @@
+class DropUsersSignInAtSignInIpSignInCount < ActiveRecord::Migration[6.1]
+  def change
+    safety_assured do
+      remove_column :users, :sign_in_count
```

Comment:
is it worth doing the reversible style of migration that has the types, etc?

Reply:
Yep, probably, will fix